### ISSUE TITLE
Avaliar o benefício de refatorar o módulo exporter.py

### ISSUE THREAD

**Issue by gustavofonseca** [scieloorg/PC-Programs#3194]
Deve ser avaliado o benefício de refatorar o módulo _exporter.py_[1] de modo que seja criada uma nova abstração que possa ser utilizada para implementar transferências locais ou remotas de arquivos.

[1] https://github.com/scieloorg/PC-Programs/blob/65e4d3e9eccdf217cfd45a12a9f899374bb54b42/src/scielo/bin/xml/app_modules/generics/exporter.py#L26

**Comment by robertatakenaka** [comment]
@gustavofonseca 
Isso está relacionado com os comentários https://github.com/scieloorg/PC-Programs/pull/3190, certo?

**Comment by robertatakenaka** [comment]
Observação: A forma que usamos o XC no servidor é uma prática somente nossa, não é o mesmo para as demais coleções. Sendo assim, para as demais coleções o ideal seria executar um ftp e não uma cópia. Eles executam o XC no Windows.

**Comment by gustavofonseca** [comment]
> @gustavofonseca
> Isso está relacionado com os comentários #3190, certo?

Sim, está relacionado ao comentário https://github.com/scieloorg/PC-Programs/pull/3190#discussion_r430072895

**Comment by joffilyfe** [comment]
> Observação: A forma que usamos o XC no servidor é uma prática somente nossa, não é o mesmo para as demais coleções. Sendo assim, para as demais coleções o ideal seria executar um ftp e não uma cópia. Eles executam o XC no Windows.

Este é um ponto muito importante, @robertatakenaka. Até o momento eu não sabia disso 😊. Eu Acho que é legal aplicarmos melhorias nos códigos que produzimos mas não sei se esse momento seria o ideal (nesta semana por exemplo).

Eu tomei a liberdade de implementar uma _ideia_ de refatoração para este módulo na sexta-feira passada. Por não estar completa e pela minha visão limitada sobre o XC, resolvi não submeter mas fica ai o [esboço](https://gist.github.com/joffilyfe/88f6f111a46f9aeb663fccdc96d835fa).
